README.md

```diff
@@ -116,10 +116,19 @@ The plugin provides two commands, `addNewSnippet` and `editSnippet`. Here is the
 code to create keymaps for them:
 
 ```lua
-vim.keymap.set("n", "<leader>se", function() require("scissors").editSnippet() end)
+vim.keymap.set("n", "<leader>se", require("scissors").editSnippet)
```

Comment:
Wrapping the `require` call in an extra function is intentional, since it makes `vim.keymap.set` lazy-load `scissors` instead of loading it at once. Could you thus please revert this?